landlab/components/landslides/landslide_probability.py

```diff
@@ -485,39 +583,77 @@ def calculate_factor_of_safety(self, i):
         self._Ksatmode = np.float32(
             self._grid.at_node["soil__saturated_hydraulic_conductivity"][i]
         )
-        self._Cmode = np.float32(self._grid.at_node["soil__mode_total_cohesion"][i])
-        self._Cmin = np.float32(self._grid.at_node["soil__minimum_total_cohesion"][i])
-        self._Cmax = np.float32(self._grid.at_node["soil__maximum_total_cohesion"][i])
+        self._Cmode = np.int32(self._grid.at_node["soil__mode_total_cohesion"][i])
+        self._Cmin = np.int32(self._grid.at_node["soil__minimum_total_cohesion"][i])
```

Comment:
I'm rather confused about why these are being cast to ints. I would expect these values to be floats.